### PR TITLE
Fix lerna config

### LINKED ISSUE #97: Remove commitHooks setting from Lerna's publish config
See: https://github.com/moxystudio/jest-config/pull/24#discussion_r437384832

### PR DESCRIPTION
**Pull request by acostalima**
## Summary

- Moved `publish` settings to `version`. It is useful for debugging purposes.
   - **NOTE:** The `publish` command supports all options provided by the `version` command.
- Added `"conventionalCommits": true` option to enable extended Conventional Commits syntax.
- Activated `conventionalcommits` preset.
- Added `ignoreChanges` option to ignore changes in files which we don't want to trigger a version bump and package release.

Closes #97

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+        "packages/*"
+    ],
+    "version": "12.1.1",
+    "command": {
+        "bootstrap": {
+            "hoist": true
+        },
+        "version": {
+            "conventionalCommits": true,
+            "changelogPreset": "conventionalcommits",
```
Comment (satazor) [inline lerna.json:12]: I think we are missing the dependency.

Comment (acostalima) [reply lerna.json:12]: Do you mean `@commitlint/config-conventional`? It's already specified in `package.json`.

Comment (satazor) [reply lerna.json:12]: Nop, I mean the one from conventional changelog. Grab it from pico.

Comment (acostalima) [reply lerna.json:12]: Done ✅

**Comment by satazor** [comment]
Looks good but tests are failing. Can you take a look?

**Comment by acostalima** [comment]
@satazor I had a look into `eslint-config-react-web-a11y`'s tests and found out the following:
- `eslint-plugin-jsx-a11y` was indeed updated to v6.3.1 somehow, which is the latest version. I've also noticed this dependency is not in `package-lock.json`. Why? Shouldn't `lerna` lock package-level dependencies?
- I suspect the test for rule `jsx-a11y/lang` is failing because of the changes introduced by https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/commit/42a2016cb52590a39974c94f1003dafdc44249af. Nothing to worry about as `PT` seems to a valid value for the `lang` attribute by conforming to known standards. Easy fix.
- I'm not sure why the test for rule `jsx-a11y/role-supports-aria-props` is failing. According to ARIA's specification, [`aria-required`](https://www.w3.org/TR/wai-aria/#aria-required) is not supported by `checkbox` role which means our test remains valid and should still pass. I will file an [issue](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/716) in the repository to understand what's going on.

**Comment by acostalima** [comment]
@satazor, I've locked `eslint-plugin-jsx-a11y` version to 6.2.3 for the time being while we wait on the follow-up of https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/716.

**Pull request merged**